frontend/src/App.js

```diff
@@ -4,6 +4,7 @@ import "./App.css";
 import Status from "./Status";
 import Home from "./Home";
 import Experiment from "./Experiment";
+import E_MIB from "./components/eMIB/E_MIB";
```

Comment:
Component names in JS are `TitleCase` with no underscores. 
I'd call it `Emib`? Just to not confuse it with a constant variable name.